about.html

```diff
@@ -2,16 +2,23 @@
 layout: default
 title: About
 ---
-<div class="code-of-conduct container">
 
-    <div class="row">
-        <div id="mentee-section" class="col-12 section">
-            <h2>About</h2>
+<div class="page-header">
+    <h1>About Us</h1>
+</div>
+<div class="container">
+    <div class="content-constrained">
+        <div class="section">
+            <h2>Our story</h2>
+            <p>Founded in 2024 in London, Women Coding Community emerged from a shared desire to create a supportive and empowering space for all women in the tech space. Our journey is one of resilience, collaboration, and empowerment, driven by the belief that together, we can shatter glass ceilings and pave the way for future generations of women in tech.</p>
         </div>
-
-
-        <div class="col-12 section global-code">
-            <h2 class="text-center">TBD</h2>
+        <div class="section">
+            <h2>Our mission</h2>
+            <p>At Women Coding Community we aim to amplify the voices, talents, and opportunities for women in the tech industry. We are committed to breaking barriers, advocating for gender equality, and creating a culture where every woman feels valued, supported, and empowered to thrive in tech.</p>
+        </div>
+        <div class="section">
+            <h2>Our community</h2>
+            <p>Diverse, vibrant, and inclusive — our community is the heart and soul of Women Coding Community. From seasoned professionals to aspiring newcomers, we welcome women of all backgrounds, experiences, and skill levels. Whether you're seeking mentorship, networking opportunities, or simply a supportive community to belong to, you'll find it here among like-minded peers who share your passion for technology.</p>
```

Comment:
Could we add our email anywhere here? or a session to contact us, something like that?